src/vrl/api/tests/test_selectielijst.py

```diff
@@ -1,3 +1,5 @@
+from dateutil.relativedelta import relativedelta
```

Comment:
can you run [isort](https://pypi.org/project/isort/) on the code? It takes care of properly sorting the imports, the configuration is in `setup.cfg`. 

Order of imports is:
1. standard lib
2. django imports
3. third party imports
4. own project imports
5. relative imports

Each section is separated by a blank line, and within the section, imports are ordered alphabetically.

Personally I've defined some aliases in my shell to automate this (to be run from the root of the repo):

```bash
which runisort 
runisort: aliased to isort --recursive --check-only --diff .

which applyisort 
applyisort: aliased to isort --recursive .
```